roles/satellite-clone/tasks/standard_backup_restore.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: Restore mongo data
+  command: "tar --selinux --overwrite -xvf {{ backup_dir }}/mongo_data.tar.gz -C /"
```

Comment:
Do you really need a separate task file for one task?

This task can be written directly in restore.yml to make it run for when: standard_backup.